src/Controller/BlogController.php

```diff
@@ -51,8 +51,6 @@ final class BlogController extends AbstractController
     #[Cache(smaxage: 10)]
     public function index(Request $request, int $page, string $_format, PostRepository $posts, TagRepository $tags): Response
     {
-        $tag = null;
```

Comment:
Please, revert this change, because tag is used on line 58 and should be defined